bin/rails

```diff
@@ -1,4 +1,27 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative '../config/boot'
-require 'rails/commands'
```

Comment:
Just noticing, but the warehouse still has bin stubs similar to this, so maybe we should apply whatever magic you did here over there as well?

Reply:
I just made https://github.com/greenriver/hmis-warehouse/pull/4057 but I won't be able to test that out.